_includes/doc/admin-guide/options/multi-line-timeout.md

```diff
@@ -0,0 +1,28 @@
+## multi-line-timeout()
+
+|  Type:|      time in seconds|
+|Default:| N/A|
+
+*Description:*  Specifies the time {{ site.product.short_name }} waits without reading new data from the source, before
+the last (potentially partial) message is flushed and sent through the pipeline as a LogMessage.
+
+Since the multi-line source detects the end of a message after finding the beginning of the subsequent message
+(indented or no-garbage/suffix mode), this option can be used to flush the last multi-line message
+in the file after a multi-line-timeout()-second timeout.
```

Comment:
Legibility:
> Since the multi-line source detects the end of a message via finding the beginning of the subsequent message
(indented or no-garbage/suffix mode), this option can be used to flush the last multi-line message
in the file after a given time.